2020-08-10 21:58:32,451 - __main__ - INFO - Start
2020-08-10 21:58:32,451 - __main__ - WARNING - Something maybe fail.
2020-08-10 21:58:32,451 - __main__ - ERROR - Faild to get result
Traceback (most recent call last):
  File "D:/python_project/try/ceshi_1.py", line 33, in <module>
    result = 10 / 0
ZeroDivisionError: division by zero
2020-08-10 21:58:32,452 - __main__ - INFO - Finished
